portfolio/src/main/webapp/script-home.js

```diff
@@ -68,3 +69,21 @@ function createListElement(text) {
   liElement.innerText = text;
   return liElement;
 }
+
+/*
+ * Creates a map using the Google Maps API.
+ */
+function initMap() {
+  const script = document.createElement('script');
```

Comment:
Why not just put this script tag directly in the HTML?

Reply:
I always thought it was a bad idea to put like inline style and script in a HTML file.

Reply:
This entire function is simply creating one single html script tag. You can erase all this code and just do:
<body>
...
...
<script src=blah defer=true async=true>
</body>

 right?

with the exception of line 82. Yo still need that in a separate js file

Reply:
I think in that case everything would have to be synchronous because of the callback parameter in the script; I'll look into it more tomorrow.

Reply:
eh, its probably fine the way you have it. i wont block this PR